sdk/test/Services/DynamoDBv2/IntegrationTests/JSONTests.cs

```diff
@@ -6,23 +6,20 @@
 
 using AWSSDK_DotNet.IntegrationTests.Utils;
 using Amazon.DynamoDBv2;
-using Amazon.DynamoDBv2.Model;
-using Amazon;
 using Amazon.DynamoDBv2.DataModel;
 using Amazon.DynamoDBv2.DocumentModel;
-using ThirdParty.Json.LitJson;
 using System.IO;
-
+using System.Text.Json;
 
 namespace AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB
 {
     public partial class DynamoDBTests : TestBase<AmazonDynamoDBClient>
     {
-        const string sampleJson = @"{
+        const string _sampleJson = @"{
 ""Name"" : ""Alan"" ,
 ""Age"" : 31,
 ""CompanyName"" : ""Big River"" ,
-""CurrentStatus"" : ""Active""
+""CurrentStatus"" : ""Active"",
```

Comment:
This test was passing before despite the missing comma, but STJ throws an exception now.